modules/siddhi-core/src/test/java/org/wso2/siddhi/core/query/window/LenghtBatchWindowTestCase.java

```diff
@@ -94,7 +94,7 @@ public void LengthBatchWindowTest2() throws InterruptedException {
                 "@info(name = 'query1') " +
                 "from cseEventStream#window.lengthBatch(" + length + ") " +
                 "select symbol,price,volume " +
-                "insert into outputStream ;";
+                "insert expired events into outputStream ;";
 
```

Comment:
Ok. Will do that. building the entire project just to ensure I haven't made
unnecessary changes elsewhere

On Thu, Jun 16, 2016 at 11:03 AM, Mohanadarshan Vivekanandalingam <
notifications@github.com> wrote:

> In
> modules/siddhi-core/src/test/java/org/wso2/siddhi/core/query/window/LenghtBatchWindowTestCase.java
> <https://github.com/wso2/siddhi/pull/163#discussion_r67289354>:
>
> > @@ -94,7 +94,7 @@ public void LengthBatchWindowTest2() throws InterruptedException {
> >                  "@info(name = 'query1') " +
> >                  "from cseEventStream#window.lengthBatch(" + length + ") " +
> >                  "select symbol,price,volume " +
> > -                "insert into outputStream ;";
> > +                "insert expired events into outputStream ;";
> >
>
> No need to close, send a commit by fixing this..
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/wso2/siddhi/pull/163/files/1461d4d148096e5993d3d1111a6b916b8e5d0f72#r67289354>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARvsJ3wSOSBtswQ9_3DGESqdHMuFXVGzks5qMOA7gaJpZM4I3AzG>
> .
>



-- 
Charini Vimansha Nanayakkara
Software Engineer at WSO2
Mobile: 0714126293